index.html

```diff
@@ -15,22 +15,59 @@
       <!-- T Stewie -->
 
       <div class="title-sun-container">
-        <div class="title">Todo</div>
+        <div class="title">TODO</div>
         <div class="sun-img-container"><img src="images/icon-sun.svg" alt=""></div>
       </div>
 
 
 
 
       <div class="input-container">
-        <input type="text" placeholder="Create a new Todo...">
+        <div class="check-img-container"><img class="empty-img" src="images/icon-check.svg" alt=""></div>
+
+        <input class="input-todo" type="text" placeholder="Create a new todo...">
       </div>
 
     </header>
 
     <section>
       <!-- Frost -->
     </section>
+
+    <section>
+      <!--T Stewie Again-->
+
+      <div class="bottom-part">
+        <div class="foot-of-todos">
+
+          <div class="items-left">
+            <span class="number-of-items-left">5</span>
+            <span>items left</span>
+          </div>
+
+          <div class="clear-completed">Clear Completed</div>
+
+        </div>
+
+        <div class="filter-section">
+          <div class="option all-option">All</div>
```

Comment:
instead of div element, you can use a button for this

```html
<button type="button" class="option all-option">All</button>
<button type="button" class="option active-option">Active</button>
 <button type="button" class="option completed-option">Completed</button>
```